apis/core/v1beta1/openstackversion_types.go

```diff
@@ -0,0 +1,195 @@
+	// TODO: add validation to cinder-operator to prevent backend being named 'default'
+	CinderVolumeImages map[string]*string `json:"cinderVolumeImages,omitempty"`
+	// ManilaShareImages custom Manila Share images for each backend (default Manila share image is stored 'default' key)
+	// TODO: add validation to cinder-operator to prevent backend being named 'default'
+	ManilaShareImages map[string]*string `json:"manilaShareImages,omitempty"`
+}
+
+// ContainerTemplate - struct that contains container image URLs for each service in OpenStackControlplane
+type ContainerTemplate struct {
+	AgentImage                    *string `json:"agentImage,omitempty"`
+	AnsibleeeImage                *string `json:"ansibleeeImage,omitempty"`
+	AodhAPIImage                  *string `json:"aodhAPIImage,omitempty"`
+	AodhEvaluatorImage            *string `json:"aodhEvaluatorImage,omitempty"`
+	AodhListenerImage             *string `json:"aodhListenerImage,omitempty"`
+	AodhNotifierImage             *string `json:"aodhNotifierImage,omitempty"`
+	ApacheImage                   *string `json:"apacheImage,omitempty"`
+	BarbicanAPIImage              *string `json:"barbicanAPIImage,omitempty"`
+	BarbicanKeystoneListenerImage *string `json:"barbicanKeystoneListenerImage,omitempty"`
+	BarbicanWorkerImage           *string `json:"barbicanWorkerImage,omitempty"`
+	CeilometerCentralImage        *string `json:"ceilometerCentralImage,omitempty"`
+	CeilometerComputeImage        *string `json:"ceilometerComputeImage,omitempty"`
+	CeilometerIpmiImage           *string `json:"ceilometerIpmiImage,omitempty"`
+	CeilometerNotificationImage   *string `json:"ceilometerNotificationImage,omitempty"`
+	CeilometerSgcoreImage         *string `json:"ceilometerSgcoreImage,omitempty"`
+	CinderAPIImage                *string `json:"cinderAPIImage,omitempty"`
+	CinderBackupImage             *string `json:"cinderBackupImage,omitempty"`
+	CinderSchedulerImage          *string `json:"cinderSchedulerImage,omitempty"`
+	DesignateAPIImage             *string `json:"designateAPIImage,omitempty"`
+	DesignateBackendbind9Image    *string `json:"designateBackendbind9Image,omitempty"`
+	DesignateCentralImage         *string `json:"designateCentralImage,omitempty"`
+	DesignateMdnsImage            *string `json:"designateMdnsImage,omitempty"`
+	DesignateProducerImage        *string `json:"designateProducerImage,omitempty"`
+	DesignateUnboundImage         *string `json:"designateUnboundImage,omitempty"`
+	DesignateWorkerImage          *string `json:"designateWorkerImage,omitempty"`
+	EdpmFrrImage                  *string `json:"edpmFrrImage,omitempty"`
+	EdpmIscsidImage               *string `json:"edpmIscsidImage,omitempty"`
+	EdpmLogrotateCrondImage       *string `json:"edpmLogrotateCrondImage,omitempty"`
+	EpdmNeutronMetadataAgentImage *string `json:"epdmNeutronMetadataAgentImage,omitempty"`
+	EdpmSriovAgentImage           *string `json:"edpmSriovAgentImage,omitempty"`
```

Comment:
This one is RELATED_IMAGE_OPENSTACK_EDPM_NEUTRON_SRIOV_AGENT_DEFAULT_IMG, so it should be EdpmNeutrionSriovAgentImage